.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -7,14 +7,15 @@ on:
     types: [published]
 
 env:
-  CIBW_BUILD: 'cp37-* cp38-* cp39-*'
+  CIBW_BUILD: 'cp37-* cp38-* cp39-* cp310-*'
+  CIBW_SKIP: "*-musllinux*"
 
   # Python build settings
   CIBW_BEFORE_BUILD: |
-    pip install pybind11 ninja && yum install -y cmake gcc gcc-c++
+    pip install pybind11 ninja
```

Comment:
How come some installs were removed?

Reply:
GCC/G++ should already be available on these images, so no need to explicitly request anymore.

Reply:
In hindsight, I think it may be better to keep these explicitly in there. Thanks for noticing.